civihr_employee_portal/src/Service/RedirectionService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Service;
+
+use Drupal\civihr_employee_portal\Helpers\WebformHelper;
+use Drupal\civihr_employee_portal\Security\PublicFirewall;
+
+class RedirectionService {
+
+  /**
+   * @var PublicFirewall
+   */
+  protected $firewall;
+
+  /**
+   * @var array
+   */
+  protected $inaccessibleRoutes = [
+    '',
+    'civicrm',
+    'civicrm/dashboard'
+  ];
+
+  public function __construct() {
+    $this->firewall = new PublicFirewall();
+  }
+
+  /**
+   * @param $user
+   * @param $route
+   * @param $requestURI
+   *
+   * @return string|null
+   *   The new route or NULL if no redirect is needed
+   */
```

Comment:
Please add param types and descriptions to the functions in this class.

Reply:
This class has been removed because of a spec change decision that redirects were not required on every page.